### PR TITLE
ISPN-12028 Improve welcome page and header

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12028

### CODE REVIEW

File: src/app/Welcome/Welcome.tsx [hunk 1]
```diff
@@ -4,25 +4,38 @@ import {
...
+  const images = {
+    lg: back5,
+    sm: back3,
+    sm2x: back4,
+    xs: back1,
+    xs2x: back2
+  };
+
   const description1 =
     'Infinispan is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```
Comment (pruivo) [inline src/app/Welcome/Welcome.tsx:40]:
```suggestion
    '{brandname} is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```

File: src/app/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -31,11 +44,15 @@ const Welcome: React.FunctionComponent<any> = props => {
...
 
-  const FooterList = () => {
+  const buildFooter = () => {
     return (
       <Stack>
         <StackItem>
-          Remotely connect to Infinispan with client applications.
+          <TextContent>
+            <Text component={TextVariants.h5}>
+              Remotely connect to Infinispan with client applications.
```
Comment (pruivo) [inline src/app/Welcome/Welcome.tsx:53]:
```suggestion
              Remotely connect to {brandname} with client applications.
```

File: src/app/Welcome/Welcome.tsx [hunk 3]
```diff
@@ -99,31 +124,36 @@ const Welcome: React.FunctionComponent<any> = props => {
...
+  );
+
   return (
     <Page>
-      <BackgroundImage src={back} />
+      <BackgroundImage src={images} filter={defaultFilter}/>
       <LoginPage
         footerListVariants={ListVariant.inline}
         brandImgSrc={icon}
         brandImgAlt="Infinispan logo"
```
Comment (pruivo) [inline src/app/Welcome/Welcome.tsx:145]: not sure if necessary...
```suggestion
        brandImgAlt="{brandname} logo"
```

File: src/app/Welcome/Welcome.tsx (hunk 1 above)
Comment (oraNod) [reply src/app/Welcome/Welcome.tsx:40]: Can we reduce it down to: {brandname} is a distributed in-memory key/value data store.

IMO optional schema and licensing is too much info for the welcome page.

File: src/app/Welcome/Welcome.tsx [hunk 4]
```diff
@@ -4,25 +4,38 @@ import {
...
+    lg: back5,
+    sm: back3,
+    sm2x: back4,
+    xs: back1,
+    xs2x: back2
+  };
+
   const description1 =
     'Infinispan is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
     'Available as an embedded Java library or as a language-independent service accessed remotely over a variety of protocols (Hot Rod, REST)';
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:41]: Suggest removing this sentence altogether. Does the console work with embedded?

File: src/app/Welcome/Welcome.tsx [hunk 5]
```diff
@@ -31,11 +44,15 @@ const Welcome: React.FunctionComponent<any> = props => {
     'Use it as a cache or a data grid. Works on bare-metal, containerized and virtualized environments. ' +
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:44]:
```suggestion
    'Use {brandname} a cache or data grid in containerized, cloud-native environments or on bare-metal. ' +
```

File: src/app/Welcome/Welcome.tsx [hunk 6]
```diff
@@ -31,11 +44,15 @@ const Welcome: React.FunctionComponent<any> = props => {
...
     'Works on AWS, Azure, Google Cloud, Kubernetes and OpenShift.';
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:45]: Feel we could drop this sentence. "Cloud-native environments" in the previous sentence covers these platforms.

File: src/app/Welcome/Welcome.tsx [hunk 7]
```diff
@@ -99,31 +124,36 @@ const Welcome: React.FunctionComponent<any> = props => {
...
     <Page>
-      <BackgroundImage src={back} />
+      <BackgroundImage src={images} filter={defaultFilter}/>
       <LoginPage
         footerListVariants={ListVariant.inline}
         brandImgSrc={icon}
         brandImgAlt="Infinispan logo"
-        backgroundImgSrc={back}
         backgroundImgAlt={brandname}
         loginTitle="Welcome to the Infinispan Server"
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:147]:
```suggestion
        loginTitle="Welcome to Infinispan Server"
```

File: src/app/Welcome/Welcome.tsx (hunk 4 above)
Comment (pruivo) [reply src/app/Welcome/Welcome.tsx:41]: the console is only for server mode.
For me, the sentence is telling me that Infinispan can be used in embedded or server mode (using hot rod/rest).
 I'm neutral if you want to remove/keep it. :)

Comment (oraNod) [reply src/app/Welcome/Welcome.tsx:41]: yeah I kind of meant that as a rhetorical question, lol. I too get the intended meaning but I'm not sure it adds benefit on the console welcome page. we do say that elsewhere. I'd advocate a more minimalist approach here.

Comment (karesti) [reply src/app/Welcome/Welcome.tsx:41]: ok

**Comment by karesti** [comment]
I did not change all the sentences because the landing page was not great at this point
Let's review the design more in another PR since I need to correct soon the fact that with keycloak we can't just land to this page without login

**Comment by karesti** [comment]
@pruivo @oraNod could you please check this again?

**Comment by pruivo** [comment]
what about https://github.com/infinispan/infinispan-console/pull/58#pullrequestreview-442335539?

File: src/app/Welcome/Welcome.tsx [hunk 8]
```diff
@@ -4,38 +4,55 @@ import {
...
+  };
+
   const description1 =
-    'Infinispan is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
+    brandname +' is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
     'Available as an embedded Java library or as a language-independent service accessed remotely over a variety of protocols (Hot Rod, REST)';
 
   const description2 =
-    'Use it as a cache or a data grid. Works on bare-metal, containerized and virtualized environments. ' +
+    'Use ' + brandname + ' a cache or data grid in containerized, cloud-native environments or on bare-metal. ' +
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:44]:
```suggestion
    'Use ' + {brandname} + ' as a cache or data grid in containerized, cloud-native environments or on bare-metal. ' +
```

File: src/app/Welcome/Welcome.tsx [hunk 9]
```diff
@@ -4,38 +4,55 @@ import {
...
 
-  const FooterList = () => {
+  const buildFooter = () => {
     return (
       <Stack>
         <StackItem>
-          Remotely connect to Infinispan with client applications.
+          <TextContent>
+            <Text component={TextVariants.h5}>
+              Remotely connect to {brandname} with client applications.
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:53]:
```suggestion
              Remotely connect to {brandname} from client applications.
```

**Comment by karesti** [comment]
> what about [#58 (review)](https://github.com/infinispan/infinispan-console/pull/58#pullrequestreview-442335539)?

missed that one!

File: src/app/Welcome/Welcome.tsx [hunk 10]
```diff
@@ -4,38 +4,55 @@ import {
...
+    lg: back5,
+    sm: back3,
+    sm2x: back4,
+    xs: back1,
+    xs2x: back2
+  };
+
   const description1 =
-    'Infinispan is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
+    brandname +' is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```
Comment (oraNod) [inline src/app/Welcome/Welcome.tsx:40]:
```suggestion
    {brandname} +' is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```

**Comment by karesti** [comment]
done an rebased @pruivo

File: src/app/Welcome/Welcome.tsx [hunk 11]
```diff
@@ -4,70 +4,99 @@ import {
...
     'Available as an embedded Java library or as a language-independent service accessed remotely over a variety of protocols (Hot Rod, REST)';
 
   const description2 =
-    'Use it as a cache or a data grid. Works on bare-metal, containerized and virtualized environments. ' +
+    'Use ' + {brandname} + ' as a cache or data grid in containerized, cloud-native environments or on bare-metal. ' +
```
Comment (pruivo) [inline src/app/Welcome/Welcome.tsx:44]: this isn't working:
`Use [object Object] as a cache or data grid in`

**Comment by karesti** [comment]
@pruivo corrected

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**